src/components/SlotSelector/SlotSelector.tsx

```diff
@@ -105,6 +104,12 @@ const SlotSelector: React.FC<SlotSelectorProps> = ({
         </IonCheckbox>
       </SectionHeader>
       <IonList className="ion-padding-bottom">
+        {group.description && (
+          <div
+            className="ion-padding-horizontal ion-padding-bottom nmdc-text-sm"
+            dangerouslySetInnerHTML={{ __html: group.description }}
```

Comment:
Update: I have an idea for an alternative. Instead of storing `group.description` as an arbitrary string, store it as a React node. Then, instead of rendering the string in the current way (i.e. via `dangerouslySetInnerHTML`), render the equivalent React node.

Here's what the definition of `uncommonGroup`, for example, would look like:

```tsx
  const uncommonGroup: SlotGroup = {
    name: "uncommon",
    description: <>
      These fields are rarely measured at the time of sample collection.
      These field are more often entered after sampling via the
      <a target="_blank" rel="noopener noreferrer" href="https://nmdc-documentation.readthedocs.io/en/latest/tutorials/submission_portal.html">
        NMDC Submission Portal
      </a>.
    </>,
    title: "Uncommon",
    slots: [],
  };
```

> The definition of `SlotGroup` may also require a change in order to accommodate this.

Here's what the rendering code would look like:

```tsx
<div className="ion-padding-horizontal ion-padding-bottom nmdc-text-sm">
    {group.description}
</div>
```

Reply:
Oh, yeah, duh that's a much better idea!